packages/kbn-test/src/jest/setup/polyfills.jsdom.js

```diff
@@ -14,3 +14,10 @@ require('whatwg-fetch');
 if (!global.URL.hasOwnProperty('createObjectURL')) {
   Object.defineProperty(global.URL, 'createObjectURL', { value: () => '' });
 }
+
+// https://github.com/jsdom/jsdom/issues/2524
+if (!global.hasOwnProperty('TextEncoder')) {
```

Comment:
this took me a long time to figure out! Had to add the TextEncoder and decoder classes to the jsdom polyfills in order to unit test openpgp calling code.  this is a known issue https://github.com/jsdom/jsdom/issues/2524.

openpgpjs uses these under the hood.